openedx_ledger/models.py

```diff
@@ -287,26 +272,16 @@ class Meta:
             "The globally unique content identifier.  Joinable with ContentMetadata.content_key in enterprise-catalog."
         )
     )
-    reference_id = models.CharField(
+    enterprise_fulfillment_uuid = models.CharField(
```

Comment:
Minor quibble, and I'll say in advance that I appreciate you trying to name this field in a way that makes it obvious what types of values it stores. :)
Naming it `enterprise_fulfillment_uuid` could make the user suspect that they're actually getting a UUID value from this field, but they'll really only get a string (which most of the time should be in the format of a uuid).  I'd either not rename this field, or leave it somewhat generic-sounding e.g. `fulfillment_identifier`.  Naming it like that also helps us keep the `Transaction` model somewhat loosely-coupled (conceptually) from enterprise-specific needs.